tests/e2e/swarm_test.go

```diff
@@ -112,7 +114,7 @@ func (s *SwarmTestSuite) TestDockerSwarm(c *C) {
 	status, host := verification.IsDockerContainerRunning(s.swarmNodes, s.serviceName, 2)
 	c.Assert(status, Equals, true, Commentf("Container %s is not running on any hosts", s.serviceName))
 
-	containerName, err = dockercli.GetContainerName(host, s.serviceName+".1")
+	containerName, err := dockercli.GetContainerName(host, s.serviceName+".1")
```

Comment:
Yes, I know this is a syntax error, but just curious why it's still passing locally and also in most CI runs. It seems very weird.